services/QuillLMS/app/workers/admin_diagnostic_reports/diagnostic_overview_worker.rb

```diff
@@ -14,10 +14,10 @@ def message
     end
 
     QUERIES = {
-      'pre-diagnostic-assigned' => PreDiagnosticAssignedQuery,
-      'pre-diagnostic-completed' => PreDiagnosticCompletedQuery,
+      'pre-diagnostic-assigned' => PreDiagnosticAssignedViewQuery,
+      'pre-diagnostic-completed' => PreDiagnosticCompletedViewQuery,
       'recommendations' => DiagnosticRecommendationsQuery,
```

Comment:
Wasn't there a rewrite of the `DiagnosticRecommendationsQuery`? Is that in a different branch or does the base class refactor speed that up?

Reply:
@dandrabik That's in #11599 , but I've just pulled that branch into this one (with attendant benchmarking) to keep things consolidated.  It may make sense for you to review it on that PR since it has only the changes relevant to that query.

Reply:
I approved it there, so consider this PR with the changes approved as well.